src/pages/product-shop/product-shop.tsx

```diff
@@ -0,0 +1,5 @@
+const ProductShop = () => {
```

Comment:
ProductShop is not good name. If it is page responsible for rendering `Products` then call it `ProductsPage` or use name which precisely describes what does it do.